library/coretests/tests/floats/mod.rs

```diff
@@ -20,6 +19,31 @@ impl Approx for f128 {
     const LIM: Self = 1e-9;
 }
 
+trait TestableFloat {
```

Comment:
Possibly we should merge `Approx` into this?

Alternatively, there could be separate floats for each logical grouping of constants. I don't really like that because it seems like a lot of boilerplate, but it would mean the traits here are more coherent.